app/client/components/flowStartMultipleProductDetailHandler.tsx

```diff
@@ -97,102 +115,122 @@ const getProductDetailSelector = (
     }
     if (sortedList.length > 1) {
       return (
-        <>
-          <PageContainer>
-            <p>
-              You have <strong>{toWords(sortedList.length)}</strong> concurrent{" "}
-              {props.productType.friendlyName}s, please select the one you would
-              like to proceed with:
-            </p>
-          </PageContainer>
-          {sortedList.map((productDetail, listIndex) => (
-            <div
-              key={productDetail.subscription.subscriptionId}
-              css={{
-                padding: "10px",
-                background:
-                  (listIndex + 1) % 2 !== 0 ? palette.neutral["7"] : undefined
-              }}
-            >
-              <PageContainer noVerticalMargin>
-                {getMainPlan(productDetail.subscription).name && (
-                  <i>({getMainPlan(productDetail.subscription).name})</i>
-                )}
-                <div css={flexCSS("flex")}>
-                  {productDetail.subscription.cancelledAt && (
-                    <span
-                      css={{
-                        fontFamily: sans,
-                        background: palette.neutral["5"],
-                        marginRight: "10px",
-                        borderRadius: "5px",
-                        padding: "2px 5px 0 6px",
-                        fontWeight: "bold"
-                      }}
-                    >
-                      CANCELLED
-                    </span>
+        <ConditionalNewLayoutWrapper
```

Comment:
any chance we could get a screenshot of the visual effect of these changes (in reply to this comment is fine)